### DISCUSSION TITLE
BOOM! One week down! Day 8!

### DISCUSSION THREAD

**Discussion by CarolineMorton** [CarolineMorton/AdventOfCode_WIR#11] [Ideas]
Well done everyone who has made it this far!

Just as a reminder to everyone, you don't have to finish the puzzles in order - they are released in order, but you can click on any of the released days to go straight to that one. I am seeing a lot of people getting stuck (here and online) on Day 6. My recommendation would be to skip it and come back to it. 

We could also collaborate on it together!

**Comment by kuhmil** [comment]
The prompt for Day 8 was so confusing to me! It seems like others in the subreddit are finding it challenging too. Having the same issue as Day 7, here is my attempt at [Part 1](https://github.com/kuhmil/AdventOfCode_WIR/tree/main/day_08)!

**Comment by AR2202** [comment]
Did this one in Claro, so not sure I can help much if anyone is stuck in Rust.

**Reply by CarolineMorton** [reply]
Feel free to share anyway. We all interested in different approaches

**Reply by AR2202** [reply]
[Day 8 ](https://github.com/AR2202/AOC2024/blob/main/other_languages/claro/day08_24.claro)in [Claro](https://docs.clarolang.com/)!

**Reply by em-baggie** [reply]
Interesting thanks for sharing, never heard of Claro

**Reply by AR2202** [reply]
A friend of mine has developed it.

**Comment by em-baggie** [comment]
<a href = "https://github.com/em-baggie/AdventOfCode_WIR/tree/main/day_08/src">My solution</a> to day 8 pt 1. This one looked hard when reading the instructions but was actually OK once I planned what to do with pseudocode. Used RustRover instead of VSC for the first time, really like it!

**Comment by DominikStefancik** [comment]
My [solution for the day 8](https://github.com/DominikStefancik/AdventOfCode_WIR/tree/main/day_08), both parts.
